demo/src/main/java/com/chartiq/demo/ui/chart/fullview/ViewOnSwipeTouchListener.kt

```diff
@@ -25,13 +25,13 @@ abstract class ViewOnSwipeTouchListener(context: Context) : View.OnTouchListener
     private inner class GestureListener : GestureDetector.SimpleOnGestureListener() {
 
         override fun onFling(
-            e1: MotionEvent,
+            e1: MotionEvent?,
             e2: MotionEvent,
             velocityX: Float,
             velocityY: Float
         ): Boolean {
             try {
-                val diffY: Float = e2.y - e1.y
+                val diffY: Float = e2.y - e1!!.y
```

Comment:
I don't understand the point of this change (and, I'm not familiar with Kotlin, but this seems potentially problematic).  If I'm reading this correctly, it seems like these changes say "e1 is allowed to contain null" and then "force e1 to not be nullable".  What is the purpose of this change and how do we know that it's safe to force non-nullable here?

Reply:
The targetSDK needs to be updated in order to be compliant for the Play Store. This Adnroid sdk function was marked uncompliant and this was the suggested change.

Reply:
Which SDK function?  I don't see any SDK code here, just ours.  And the logic seems problematic.  Explicitly making e1 nullable and then asserting that it will not be null without any associated guard seems wrong and very suspicious as a "suggested change".

Reply:
This is overriding the onFling method in the Android sdk. The first parameter being nullable was added in SDK 34. What is being done here seems to be an acceptable solution for the few questions on stackoverflow and such. 

https://kotlinlang.org/docs/null-safety.html#not-null-assertion-operator
https://stackoverflow.com/questions/73463685/gesturedetector-ongesturelistener-overridden-methods-are-not-working-in-android
https://github.com/capacitor-community/photoviewer/issues/56

Reply:
Thanks, that answers part re: the SDK, but I don't see anywhere recommending the non-null assertion.  It's the combination of `?` + `!!` that stands out the most.  If e1 is ever null, line 34 will throw an NPE.  Are we ok with that?